smif/sos_model.py

```diff
@@ -872,20 +884,17 @@ def add_scenario_data(self, data):
         Expect a dictionary, where each key maps a parameter
         name to a list of data, each observation with:
 
-        - timestep
         - value
         - units
+        - timestep (must use a timestep from the SoS model timesteps)
         - region (must use a region id from scenario regions)
         - interval (must use an id from scenario time intervals)
 
-        Add a dictionary of list of :class:`smif.SpaceTimeValue` named
-        tuples,
-        for ease of iteration::
+        Add a dictionary of :class:`numpy.ndarray`
 
-                data[year][param] = SpaceTimeValue(region, interval, value, units)
+                data[param] = np.empty((num_timesteps, num_intervals, num_regions))
```

Comment:
Safer to use `np.zeros` (really), particularly when filling of arrays is deferred...